### ISSUE TITLE
Specify dose units in edknrc

### ISSUE THREAD

**Issue by mchamberland** [nrc-cnrc/EGSnrc#1032]
[edknrc cavity dose output does not indicate the units of the results.](https://github.com/nrc-cnrc/EGSnrc/blob/9dca54412c15dd5188193eb51feb9de59b7d2c36/HEN_HOUSE/user_codes/edknrc/edknrc.mortran#L1864) In fact, neither the code documentation or PIRS-702 mention the units of any of the outputs or whether the results are normalized.

On a related note, [line 3836](https://github.com/nrc-cnrc/EGSnrc/blob/9dca54412c15dd5188193eb51feb9de59b7d2c36/HEN_HOUSE/user_codes/edknrc/edknrc.mortran#L3836) of the source code indicates "dose per incident particle", with units of "Gy", but I wonder if it would be better to explicitly specify the units as "Gy per incident particle". Leaving it as "Gy" is ambiguous and may lead to confusion as to the results. But perhaps that's a wider discussion to be had across EGSnrc?

**Comment by ftessier** [comment]
Agreed that the units ought to be spelled out everywhere. I am fine with `Gy` units for "dose per incident particle", but it would do no harm to state that explicitly too.
